tools/telemetry-generator/package.json

```
@@ -25,7 +25,7 @@
 		"prettier": "prettier --check . --cache --ignore-path ../../.prettierignore",
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../.prettierignore",
 		"start": "node bin/run",
-		"test": "echo \"Error: no test specified\" && exit 1",
+		"test": "mocha --recursive \"dist/test/**/*.test.*js\" --exit",
```

That flag is not the same as the `exit 1` from before, it shouldn't be present in our packages (the ones that do have it have issues and we should eventually address them to remove the flag).
```suggestion
		"test": "mocha --recursive \"dist/test/**/*.test.*js\"",
```